### PR TITLE
Icons: Tools and Construction

### PR DESCRIPTION
**Pull request by Andreto**
**So far:** Axe, Gavel, Hammer, Hardhat, Shovel
image
I would like to add a screwdriver but I couldn't really get it right.

**Satisfies Requests:**
Gavel: https://github.com/feathericons/feather/issues/59
Hammer: https://github.com/feathericons/feather/issues/717

### CODE REVIEW

**Comment by ericfennis** [comment]
Can you try to match it with the `tool` icon?

https://lucide.netlify.app/icon/tool

**Comment by Andreto** [comment]
@ericfennis  Something like this?
image

**Comment by ericfennis** [comment]
@Andreto Looks awesome!
I think it's better if we rename `tool` to wrench.

File: icons/hammer.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="M15 12l-8.5 8.5c-.83.83-2.17.83-3 0h0a2.12 2.12 0 010-3L12 9"/>
+    <path d="M18 56l4-4M21 53l-1.15-1.15A2.91 2.91 0 0119 49.8V49l-2.5-2.5a5.11 5.11 0 00-3.62-1.5h-2.82l.85.75a5.66 5.66 0 011.9 4.24v1.43l1.84 1.83h2.27L19 55" transform="matrix(1.0889 0 0 1.0909 -1.96 -46.1)"/>
```
Comment (ericfennis) [inline icons/hammer.svg:13]: Can you remove the transform matrix?

File: icons/hardhat.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M2 18a1 1 0 001 1h18a1 1 0 001-1v-2a1 1 0 00-1-1H3a1 1 0 00-1 1v2zM10 10V5a1 1 0 011-1h2a1 1 0 011 1v5M4 15v-3a6 6 0 016-6M14 6a6 6 0 016 6v3"/>
```
Comment (ericfennis) [inline icons/hardhat.svg:12]: Can you break up the path in separate elements, some programs don't like merged paths you get weird artifacts

File: icons/shovel.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M2 22v-5l5-5 5 5-5 5zM9.5 14.5L16 8M17 2l5 5-.5.5a3.53 3.53 0 01-5 0h0a3.53 3.53 0 010-5L17 2"/>
```
Comment (ericfennis) [inline icons/shovel.svg:12]: same here

**Review by ericfennis — CHANGES_REQUESTED** [review]

File: icons/hammer.svg (hunk 1 above)
Comment (Andreto) [reply icons/hammer.svg:13]: Maybe, it wasn't as easy as just removing the tag so but I think that there are some options in Inkscape to remove it. I'll try it later today.

File: icons/hardhat.svg (hunk 2 above)
Comment (Andreto) [reply icons/hardhat.svg:12]: That was what I got out of SVGOMG. Are there any settings for "Combine Paths"?

**Comment by Andreto** [comment]
@ericfennis I managed to fix the mentioned issues by skipping SVGOMG all together and striping down the XML myself. Will commit soon but first I just wondered, should it be `hardhat` or `hard-hat`?

File: icons/hammer.svg (hunk 1 above)
Comment (ericfennis) [reply icons/hammer.svg:13]: Cool!

File: icons/hardhat.svg (hunk 2 above)
Comment (ericfennis) [reply icons/hardhat.svg:12]: yeah there is but that only works one way (multiple paths => one path)

**Comment by ericfennis** [comment]
Nice work! I think we should use the correct spelling, And that is two words, so `hard-hat` it is.

File: icons/hammer.svg [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m15 12-8.501 8.501c-0.828 0.828-2.17 0.828-2.998 0-1e-3 -1e-3 -1e-3 -1e-3 -2e-3 -0-0.828-0.828-0.828-2.17 0-2.998l8.501-8.501"/>
+  <path d="m17.644 15 4.3557-4.3637"/>
+  <path d="m20.911 11.7s-0.65662-0.6578-1.249-1.2512c-0.59455-0.5957-0.92885-1.403-0.92885-2.2451v-0.8673s-1.4886-1.4913-2.7256-2.7306c-1.0432-1.045-2.4599-1.633-3.9375-1.633h-3.0697s0.40508 0.36 0.91796 0.8149c1.3241 1.1738 2.082 2.8581 2.082 4.6265v1.5589l1.9993 1.9997h2.4664l2.2671 1.9091"/>
```
Comment (ericfennis) [inline icons/hammer.svg:14]:
```suggestion
  <path d="M15 12l-8.5 8.5c-.83.83-2.17.83-3 0 0 0 0 0 0 0a2.12 2.12 0 010-3L12 9" />
  <path d="M17.64 15L22 10.64" />
  <path d="M20.91 11.7l-1.25-1.25c-.6-.6-.93-1.4-.93-2.25v-.86L16.01 4.6a5.56 5.56 0 00-3.94-1.64H9l.92.82A6.18 6.18 0 0112 8.4v1.56l2 2h2.47l2.26 1.91" />
```

File: icons/gavel.svg [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m14 13-7.501 7.501c-0.828 0.828-2.17 0.828-2.998 0-1e-3 -1e-3 -1e-3 -1e-3 -2e-3 -0-0.828-0.828-0.828-2.17 0-2.998l7.501-7.501"/>
+  <path d="m16 16 6-6"/>
+  <path d="m8 8 6-6"/>
+  <path d="m9 7 8 8"/>
+  <path d="m21 11-8-8"/>
```
Comment (ericfennis) [inline icons/gavel.svg:16]:
```suggestion
  <path d="M14 13l-7.5 7.5c-.83.83-2.17.83-3 0 0 0 0 0 0 0a2.12 2.12 0 010-3L11 10" />
  <path d="M16 16l6-6" />
  <path d="M8 8l6-6" />
  <path d="M9 7l8 8" />
  <path d="M21 11l-8-8" />
```

File: icons/hard-hat.svg [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m2.0 18.0c0 0.265 0.105 0.52 0.293 0.707 0.187 0.188 0.442 0.293 0.707 0.293h18c0.265 0 0.52-0.105 0.707-0.293 0.188-0.187 0.293-0.442 0.293-0.707v-2c0-0.265-0.105-0.52-0.293-0.707-0.187-0.188-0.442-0.293-0.707-0.293h-18c-0.265 0-0.52 0.105-0.707 0.293-0.188 0.187-0.293 0.442-0.293 0.707v2z"/>
+  <path d="m10.0 10.0v-5c0-0.265 0.105-0.52 0.293-0.707 0.187-0.188 0.442-0.293 0.707-0.293h2c0.265 0 0.52 0.105 0.707 0.293 0.188 0.187 0.293 0.442 0.293 0.707v5"/>
+  <path d="m4.0 15.0v-3.002c0-1.591 0.632-3.116 1.757-4.241s2.65-1.757 4.241-1.757h2e-3"/>
+  <path d="m14.0 6.0h2e-3c1.591 0 3.116 0.632 4.241 1.757s1.757 2.65 1.757 4.241v3.002"/>
```
Comment (ericfennis) [inline icons/hard-hat.svg:15]:
```suggestion
  <path d="M2 18a1 1 0 001 1h18a1 1 0 001-1v-2a1 1 0 00-1-1H3a1 1 0 00-1 1v2z" />
  <path d="M10 10V5a1 1 0 011-1h2a1 1 0 011 1v5" />
  <path d="M4 15v-3a6 6 0 016-6h0" />
  <path d="M14 6h0a6 6 0 016 6v3" />
```

File: icons/shovel.svg [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m2.0 22.0v-5l5-5 5 5-5 5z"/>
+  <path d="m9.5 14.5 6.5-6.5"/>
+  <path d="m17.0 2.0 5 5-0.501 0.501c-0.663 0.663-1.562 1.035-2.499 1.035s-1.836-0.372-2.499-1.035c-1e-3 -1e-3 -1e-3 -1e-3 -2e-3 -0-0.663-0.663-1.035-1.562-1.035-2.499s0.372-1.836 1.035-2.499l0.501-0.501"/>
```
Comment (ericfennis) [inline icons/shovel.svg:14]:
```suggestion
  <path d="M2 22v-5l5-5 5 5-5 5z" />
  <path d="M9.5 14.5L16 8" />
  <path d="M17 2l5 5-.5.5a3.53 3.53 0 01-5 0s0 0 0 0a3.53 3.53 0 010-5L17 2" />
```

File: icons/ruler.svg [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="m16.0 2.0 6 6-14 14-6-6 14-14"/>
+    <path d="m7.5 10.5 2 2"/>
+    <path d="m10.5 7.5 2 2"/>
+    <path d="m13.5 4.5 2 2"/>
+    <path d="m4.5 13.5 2 2"/>
```
Comment (ericfennis) [inline icons/ruler.svg:16]:
```suggestion
    <path d="m16.0 2.0 6 6-14 14-6-6 14-14" />
    <path d="m7.5 10.5 2 2" />
    <path d="m10.5 7.5 2 2" />
    <path d="m13.5 4.5 2 2" />
    <path d="m4.5 13.5 2 2" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Nice, the design of the tools are awesome! 🚀

I don't know if you already used SVGOMG to minify the paths, I set the precision most of the time on 2 (reduces the decimals in the paths) most of the time no difference is seen and it reduces the files size drastically.

I minified them a bit for you.

**Comment by Andreto** [comment]
Nice, I committed these changes.

**Comment by ericfennis** [comment]
@Andreto Nice work!

**Pull request merged**